lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -320,12 +320,15 @@ def create_tar(cv, cvv, repositories, json)
         export_file = "#{export_prefix}.json"
         export_repos_tar = "#{export_prefix}-repos.tar"
         export_tar = "#{export_prefix}.tar"
+        # rubocop:disable Style/UnneededInterpolation
+        export_dir = File.expand_path("#{options['option_export_dir']}")
```

Comment:
Why disable rubocop here instead of following the rule?

```suggestion
        export_dir = File.expand_path(options['option_export_dir'])
```